src/databases/Uintah/CMakeLists.txt

```diff
@@ -1,7 +1,15 @@
+#
+# NOTE: THIS FILE HAS HAND MODIFICATIONS !!! 
+#
 # DO NOT EDIT THIS FILE! THIS FILE IS AUTOMATICALLY GENERATED BY xml2cmake
 PROJECT(Uintah_database)
 
 INCLUDE(${VISIT_SOURCE_DIR}/CMake/PluginMacros.cmake)
+#
+# NOTE: THIS FILE HAS HAND MODIFICATIONS !!! 
```

Comment:
Since you are already visiting all these cases, why not just add solution for #4106 and use that here as opposed to these comments?

Reply:
I just wanted to try to capture what could not be wired up during this pass.

For the two cases we have for database plugins that can't be gen'ed they are actually minor.  My suggestion would be to not maintain manual edits at all for these cases, but to rework to make sure they could be code-gened.